src/api/mapping.ts

```diff
@@ -93,11 +94,11 @@ mappingRouter.get('/', (req, res) => {
 
 mappingRouter.delete('/:id', async (req, res) => {
   const domains = getMappings()
-  const deletedDomain = domains.find(e => e.id === req.params.id)
-  const updatedDomains = domains.filter(e => {
-    return e.id !== req.params.id
-  })
-  setData('mappings', updatedDomains)
+  const deletedDomain = getMappingFromId(req.params.id)
+  const updatedDomains = { ...domains }
+  delete updatedDomains[deletedDomain.fullDomain]
+  // Setting data as array to retain data structure of existing sites
+  setData('mappings', Object.values(updatedDomains))
```

Comment:
If this line is not done, it will be incompatible with already launched myproxy servers after upgrading.  

When I was testing on my droplet, the create and delete servers functionality did not work correctly since the mappings object in data.db is not in the proper format.